style.css

```diff
@@ -148,6 +148,434 @@ body {
   cursor: pointer;
 }
 
+.middle-section {
+  min-height: 281px;
+  padding-bottom: 51px;
+  background: #FAE8E4;
+}
+
+.middle-section__image {
+  height: 264px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  overflow: hidden;
+  margin-top: 62px;
+  padding: 0;
+  margin: 0;
+}
+
+.middle-section__img {
+  margin-top: 14px;
+}
+
+.middle-section__img img {
+  width: 100%;
+  height: 100%;
+  -o-object-fit: contain;
+  object-fit: contain;
+}
+
+.middle-section__clutch {
+  padding: 15px 30px;
+  margin: 0 28px 0;
+  background: #FFFFFF;
+  -webkit-box-shadow: 11px 25px 40px -34px rgba(0,0,0,0.8);
+  box-shadow: 11px 25px 40px -34px rgba(0,0,0,0.8);
+}
+
+.middle-section__title,
+.middle-section__subtitle {
+  margin: 0;
+  margin-top: 3px;
+}
+
+.middle-section__description {
+  margin-top: 13px;
+  line-height: 21px;
+  opacity: .5;
+}
+
+.middle-section__btn_busket:hover {
+  cursor: pointer;
+  background-color: #fae8e4;
+}
+
+.quantity,
+.middle-section__btn_other {
+  display: none;
+}
+
+.middle-section__form-colors {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  width: 133px;
+  height: 30px;
+  padding: 0;
+  margin-top: 5px;
+}
+
+.form__colors-btn {
+  display: none;
+}
+
+.form__colors-item {
+  position: relative;
+  display: inline-block;
+  width: 29px;
+  height: 29px;
+}
+
+.form__colors-item::before {
+  content: "";
+  position: absolute;
+  left: 3px;
+  top: 0;
+  display: block;
+  width: 23px;
+  height: 23px;
+  background: #DEDEDE;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+input[type="radio"]:checked + .form__colors-item::before {
+  left: 0;
+  width: 29px;
+  height: 29px;
+  -webkit-transition: all 300ms ease-in-out;
+  -o-transition: all 300ms ease-in-out;
+  transition: all 300ms ease-in-out;
+  -webkit-transform: translate(0,-9%);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)